### PR TITLE
TST: add tests for Timestamp.toordinal/fromordinal

### PR DESCRIPTION
**Pull request by sinhrks**
- [x] follow-up for #13593
- [x] tests added / passed
- [x] passes `git diff upstream/master | flake8 --diff`

added tests and fixed missed doc change.

### CODE REVIEW

File: pandas/tslib.pyx [hunk 1]
```diff
@@ -235,7 +235,7 @@ class Timestamp(_Timestamp):
     ----------
     ts_input : datetime-like, str, int, float
         Value to be converted to Timestamp
-    offset : str, DateOffset
+    freq : str, DateOffset
```
Comment (jreback) [inline pandas/tslib.pyx]: let's list offset (and say it's deprecated)

Comment (jreback) [inline pandas/tslib.pyx]: as well

Comment (sinhrks) [inline pandas/tslib.pyx]: updated docstrings of class and ``fromordinal``.

**Comment by jreback** [comment]
thanks!

**Pull request closed without merging**